FidoCore.md

```diff
@@ -1003,7 +1003,7 @@ Example:
   [RequestMiddleware('First decorator')]
```

Comment:
You could add a simple example of how to use the comma separated parameters.
```suggestion
  [RequestMiddleware('First decorator')]
  [RequestMiddleware('First decorator with params', '1, 2, 3')]
```

Reply:
That is planned in the future